src/test_sort_cards.py

```diff
@@ -0,0 +1,27 @@
+from random import randint, choice
+
+SORTED_CARDS = 'A23456789TJQK'
+TEST_CARDS = (
+    'A', '3', 'T', 'T824Q', 'QKJ6932', 'J69327A8', 'J679J7327A8',
+    'TA8AAA24AQA', 'AAAAAAAAAAAAA', '39A5T824Q7J6K', 'Q286JK395A47T',
+    '54TQKJ69327A8', 'Q286JK395A47TQ286JK395A47T',
+    'Q286JKKKKK395AAA47TQ286JK395A47T',
+    'AAAAAAAAAAAAAQ286JK395A47TQ286JK395A47T'
+)
+
+
+def solution(cards):
+    return sorted(cards, key='A23456789TJQK'.index)
+
+
```

Comment:
these tests should be compatible with pytest.  They do not run in their current state.